docs/privacyidea.md

```diff
@@ -216,6 +216,41 @@ Use the following example:
         'excludeClientIPs'  => array("10.0.0.0-10.2.0.0", "192.168.178.2"),
     ),
 
+    /**
+     *  This filter is optional, if you want to disable clients by entityID, you should enable it.
+     */
+    22 => array(
+        'class'                => 'privacyidea:checkEntityID',
+
+        /**
+         *  Specify setPath and setKey to equal enabledPath and enabledKey
+         */
+        'setPath'              => 'privacyIDEA',
+        'setKey'               => 'enabled',
+
+        /**
+         *  For a match, the variable $state[$setPath][$setPath] will be set to this value.
```

Comment:
It is not clear to me, for what this is.
Would it have to be ``$state[$setPath][$setKey``, e.g.

    $state['privacyIDEA']['enabled'] = false

So could you explain: Does this need to be set to `false` so that the below mentioned entityIDs do not require a 2nd factor?
I.e. setting this to `true`  would result in roughly no effect?

Reply:
on match, the filter will set $state['privacyIDEA']['enabled'] to the value of onmatch. By setting onmatch = false, a 2fa blacklist is realized, onmatch = true is intended to setup a whitelist, but you are right. This does not work.

Reply:
I will change the code to reverse the matching and adjust the documentation.